mixpanel/__init__.py

```diff
@@ -15,7 +15,7 @@
 import urllib
 import urllib2
 
-VERSION = '4.0.2'
+__version__ = '4.0.2'
```

Comment:
This changes a public API, so you might want to either preserve `VERSION = __version__` or make a change to the major release